src/cmds/qmgr.c

```diff
@@ -2211,25 +2211,29 @@ display(int otype, int ptype, char *oname, struct batch_status *status,
 				}
 			}
 			else  {
+				indent_len = 4;
 				if (otype == MGR_OBJ_RSC) {
 					if ((attr != NULL) && (strcmp(attr->name, "type") == 0)) {
 						struct resc_type_map *rtm = find_resc_type_map_by_typev(atoi(attr->value));
 						if (rtm) {
-							printf("\ttype = %s\n", rtm->rtm_rname);
+							printf("%*s", indent_len, " ");
+							printf("type = %s\n", rtm->rtm_rname);
```

Comment:
it was getting a bit confusing because of the "%*s" as that single formatter requires two arguments (the length and the blank space). So, i thought it'd be more clear if I separated it out. But if you think it doesn't make much difference, I'll combine them into one.